Makefile

```diff
@@ -40,9 +40,8 @@ PERSISTENCE_DRIVER ?= cassandra
 TEMPORAL_DB ?= temporal
 VISIBILITY_DB ?= temporal_visibility
 
-ifdef TEST_TAG
-override TEST_TAG := -tags $(TEST_TAG)
-endif
+BUILD_TAG := -tags protolegacy
+TEST_TAG := $(BUILD_TAG),$(TEST_TAG)
```

Comment:
if we actually get a TEST_TAG then this won't work without override (afaik, I just read the docs).
how about
```suggestion
# Always use "protolegacy" tag to allow disabling utf-8 validation on proto messages
# during proto library transition.
ALL_BUILD_TAGS := protolegacy,$(BUILD_TAG)
ALL_TEST_TAGS := $(ALL_BUILD_TAGS),$(TEST_TAG)
BUILD_TAG_FLAG := -tags $(ALL_BUILD_TAGS)
TEST_TAG_FLAG := -tags $(ALL_TEST_TAGS)

# ... use $(BUILD_TAG_FLAG) and $(TEST_TAG_FLAG) below ...
```

Reply:
Good catch. I've committed this change